docs/src/site/twiki/restapi/ExtensionDescription.twiki

```diff
@@ -0,0 +1,24 @@
+---++  GET api/extensions/describe/:extension­name
+   * <a href="#Description">Description</a>
+   * <a href="#Parameters">Parameters</a>
+   * <a href="#Results">Results</a>
+   * <a href="#Examples">Examples</a>
+
+---++ Description
+Description of an extension.
+
+---++ Parameters
+   * :extension­name Name of the extension.
+
+---++ Results
+Outputs the README of the specified extension.
+
+---++ Examples
+---+++ Rest Call
+<verbatim>
+GET http://localhost:15000/api/extensions/describe/hdfs­mirroring
+</verbatim>
```

Comment:
Extension APIs start with "api/extension/..." instead of "api/extensions/...". So for other extension repository APIs.